src/App.jsx

```diff
@@ -14,57 +15,60 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [message, SetMessage] = useState('Jam is selected');
```

Comment:
The setter function `SetMessage` should be `setMessage` to follow JavaScript naming conventions for hooks.